src/renderer/src/views/Home/index.tsx

```diff
@@ -34,37 +39,82 @@ const BottomBarGridSection = styled.div(`
 
 export type PanelName = 'territory' | 'observations' | 'sync' | 'settings'
 
-interface Props {
-  showBottomBar?: boolean
+type LocationState = {
+  showSnackBar?: boolean
 }
 
-type LocationState = { defaultTab: PanelName }
```

Comment:
we always default to observation, so this was unnecessary

Reply:
that was added so we can support directly navigating to the `Settings` tab (one of the options in the initial landing page)

Reply:
i don't see it anywhere in the codebase (just using vscode search). Is it in an open PR?

Reply:
yeah i believe it's being used in https://github.com/digidem/mapeo-desktop-shell/pull/59, although that's using it via the menu bar. looks like we haven't addressed navigating to the Settings invite flow from the landing page yet